Makefile

```diff
@@ -523,15 +495,15 @@ PUBLISH_BINTRAY_REDHAT += \
 endif
 
 ifeq ($(TARGET_PLATFORM),win32)
-electron-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
+electron-installer-portable: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH)-portable.exe
 electron-installer-nsis: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe
 cli-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
 TARGETS += \
-	electron-installer-zip \
+	electron-installer-portable \
```

Comment:
We do, but I've now realised that they have a separate `electron-installer-appimage` Makefile target, so the removal of `electron-installer-zip` is fine :)